lua/terrortown/menus/gamemode/administration/hud.lua

```diff
@@ -71,6 +72,20 @@ function CLGAMEMODESUBMENU:Populate(parent)
 			end
 		})
 	end
+
+	local form3 = vgui.CreateTTT2Form(parent, "header_hud_toggleable")
+
+	for i = 1, #hudElemList do
+		local elem = hudElemList[i]
+
+		if not elem.togglable then continue end
```

Comment:
is that correct? not toggl e able? Both is possible, just noticed. Dont know which one is used in code

Reply:
![image](https://user-images.githubusercontent.com/13639408/141679832-a18a40ba-0ba0-4a27-9823-01f90b7ad830.png)

yes